### PR TITLE
update versioning doc

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

- Use heading+paragraph instead of table to display the two release series. While TiDB has **two** release series, the table has **three** rows, which might be confusing. Because patch release is only available for LTS, we can put patch release under the LTS heading.
- Rewrite the opening paragraph and add a note to avoid confusion about whether v5.1~v5.4 is LTS.
- Use `<Important>` shortcode.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/versioning.md [hunk 1]
```diff
@@ -5,34 +5,61 @@ summary: 了解 TiDB 版本发布的规则。
...
 ## 版本命名
 
-TiDB 版本的命名方式为 `X.Y.Z`。
+TiDB 版本的命名方式为 `X.Y.Z`。`X.Y` 代表一个版本系列。
 
-- 从 TiDB 1.0 起，`X` 每年依次递增并引入新的功能和改进。
-- `Y` 从 0 开始依次递增，Y 的递增代表引入新的功能和改进。`X.Y` 代表一个版本系列。
+- 从 TiDB 1.0 起，`X` 每年依次递增，X 的递增代表引入新的功能和改进。
+- `Y` 从 0 开始依次递增，Y 的递增代表引入新的功能和改进。
 - `Z` 代表补丁版本的编号，一个版本系列第一次发版时 `Z` 默认为 0，补丁版本的 `Z` 从 1 开始依次递增。
```
Comment (qiancai) [inline releases/versioning.md:25]:
```suggestion
- 一个版本系列第一次发版时 `Z` 默认为 0，后续发补丁版本时 `Z` 从 1 开始依次递增。
```

Comment (qiancai) [reply releases/versioning.md:25]: 建议修改原因：`Z` 为 0 时不代表补丁版本的编号

File: releases/versioning.md [hunk 2]
```diff
@@ -5,34 +5,61 @@ summary: 了解 TiDB 版本发布的规则。
...
 
-> **重要：**
->
-> 建议始终升级到当前系列的最新补丁版本。
+<Important>
+建议始终升级到当前系列的最新补丁版本。
+</Important>
 
-从 TiDB 6.0.0 开始，TiDB 引入一个新的版本系列 DMR，并与 LTS 版共同作为 TiDB 的两个版本系列：
+TiDB 有两个版本系列：
```
Comment (qiancai) [inline releases/versioning.md:12]:
```suggestion
TiDB 提供两个版本系列：
```

**Review by Ivy-YinSu — APPROVED** [review]

File: releases/versioning.md [hunk 3]
```diff
@@ -5,34 +5,61 @@ summary: 了解 TiDB 版本发布的规则。
...
 
-- 长期支持版本 (Long-Term Support Releases, LTS)
-- 开发里程碑版本 (Development Milestone Releases, DMR)
+- 长期支持版本
+- 开发里程碑版本（自 TiDB v6.0.0 起引入）
 
-TiDB 5.0.0 及其之前的版本规则，请查看[历史版本](#不再沿用的历史版本号)。
+TiDB v5.0.0 及其之前的版本规则，请查看[历史版本](#不再沿用的历史版本号)。
```
Comment (qiancai) [inline releases/versioning.md:17]: 这句话看起来和版本命名相关，是否可以放到 “版本命名” 呢？

Comment (ran-huang) [reply releases/versioning.md]: All comments addressed. PTAL again. Thanks!

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**